src/OpenTelemetry/Resources/ResourceBuilderExtensions.cs

```diff
@@ -89,7 +95,20 @@ public static class ResourceBuilderExtensions
         /// <summary>
         /// Adds service information to a <see cref="ResourceBuilder"/>
         /// following <a
-        /// href="https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk">semantic
+        /// href="https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#service">semantic
+        /// conventions</a>.
+        /// </summary>
+        /// <param name="resourceBuilder"><see cref="ResourceBuilder"/>.</param>
+        /// <returns>Returns <see cref="ResourceBuilder"/> for chaining.</returns>
+        public static ResourceBuilder AddDefault(this ResourceBuilder resourceBuilder)
```

Comment:
this method can be a private helper in the ResourceBuilder class. No need to exposing as public API, as users already have `CreateDefault`

Reply:
I'll remove the addition of this to the API as well right? Looks like after all these changes, we no longer have anything new that's callable.